### ISSUE TITLE
网络结构疑问

### ISSUE THREAD

**Issue by LC3096** [lyh983012/SNN-ANN-Pretrain#1]
网络结构中的所有最大池化层均被取消，是出于什么考虑

**Comment by lyh983012** [comment]
resnet结构中么？只是因为残差层中不需要最大池化，首层还是用了的。另外binary SNN中用最大池化，如果输入不够稀疏，信息损失会非常严重，一旦脉冲发放过于稠密，有可能池化后变为近似全1的稠密矩阵

**Comment by LC3096** [comment]
位于exp1的网络定义中，
def __init__()
   self.conv1 = nn.Sequential(
            Snn_Conv2d(3, 64*k, kernel_size=7, padding=3, bias=False, stride=2),
            batch_norm_2d(64*k),
            )
   self.pool = nn.MaxPool2d(kernel_size=3, stride=2, padding=1)
def forward():
  output = self.conv1(input)
  output = self.mem_update(output)
  output = self.conv2_x(output)

## 首层的self.pool也被舍去了

**Comment by lyh983012** [comment]
用stride替代也可以，都只是为了压缩特征图尺寸。在静态图片输入里spike一般都相对稠密，不太建议直接pooling
